.travis.yml

```diff
@@ -36,28 +36,17 @@ jobs:
   include:
     - stage: lint
       name: "Linting"
-      python: "3.6"
+      python: "3.7"
       before_script:
         - travis_retry pip install -e ".[dev-lint]"
       script:
         - make lint
 
     - stage: rst
       name: "RST (README.rst + docs) syntax check"
-      python: "3.6"
+      python: "3.7"
       before_script:
         - travis_retry pip install -e ".[dev-docs]"
       script:
         - make rst_check
 
-    # The `test` stage using the `python` matrix above is included implicitly.
```

Comment:
Could you please return this comment back?

The rest of the diff looks good to me, thank you!

Reply:
ah right, the comment is still valid and helpful